packages/server/modules/core/graph/resolvers/streams.ts

```diff
@@ -369,16 +385,40 @@ export = {
     }
   },
   LimitedUser: {
-    async streams(parent, args, context) {
+    async streams(parent, args, ctx) {
       // a little escape hatch for admins to look into users streams
+      // const forOtherUser = parent.id !== ctx.userId
+      const isAdmin = adminOverrideEnabled() && ctx.role === Roles.Server.Admin
+      const forOtherUser = !isAdmin
```

Comment:
Is this correct? Only show other user's streams if **not** an admin? Surely it's the inverse?